views/layout.html

```diff
@@ -0,0 +1,124 @@
+{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the application. -#}
+{%- set assetPath = '/govuk/assets' -%}
+
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
```

Comment:
i would also remove the imports we don't need, we can add more when we need to. Sorry!

Reply:
How do I know which ones are and aren't needed? I've lifted this from the prototype, so strikes me as easier to just leave this as is, or, as more prototype screens are added, developers will have to come back to layout.html, work out what imports are missing and add them in.

Reply:
you can tell from what is used in each page, govUkInput etc.. We can leave for another PR. It's best to keep to a minimum or we increase client load times if code that is not needed is loaded